controllers/tables/index.js

```diff
@@ -111,19 +117,30 @@ exports.remove = function *removeTable() {
 /**
  * Get Tables
  * @deprecated
- */
+*/
```

Comment:
Is this change was required by ESLint? Generally jsdocs should be generated like that, but I would prefer to have `*` symbols line by line alligned